frontend/src/static/js/components/playlists-selection/PlaylistsSelection.jsx

```diff
@@ -98,14 +98,18 @@ export function PlaylistsSelection(props) {
     let ret = [];
     let i = 0;
     while (i < playlists.length) {
```

Comment:
@mgogoulos media_list is undefined for playlists with no media. This causes issues anytime this function is called and makes the playlist functionality non deterministic